### PR TITLE
TINY-4857: Fixed the `media` plugin embed content not being processed safely in some cases

### PR DESCRIPTION
**Pull request by lnewson**
This was caused by our EME logic creating a DOM fragment, which meant the HTML was loading into an element and executed. I've updated the EME logic to also run via the SaxParser logic so it's not inserted into the dom until it's sanitized when it gets inserted into the editor.

Note: It unfortunately has some mutation, but I didn't want to rewrite too much which could break the older processing logic given this is for a patch release.

### CODE REVIEW

File: modules/tinymce/src/plugins/media/main/ts/core/Sanitize.ts [hunk 1]
```diff
@@ -38,16 +38,19 @@ const sanitize = function (editor: Editor, html: string) {
     start (name, attrs, empty) {
       blocked = true;
 
-      if (name === 'script' || name === 'noscript') {
+      if (name === 'script' || name === 'noscript' || name === 'svg') {
```
Comment (metricjs) [inline modules/tinymce/src/plugins/media/main/ts/core/Sanitize.ts:41]: Why this change?

Comment (metricjs) [reply modules/tinymce/src/plugins/media/main/ts/core/Sanitize.ts:41]: Or the change below to delete attrs?

Comment (metricjs) [reply modules/tinymce/src/plugins/media/main/ts/core/Sanitize.ts:41]: Is it to work around the Schema weirdness you found with videos?

Comment (lnewson) [reply modules/tinymce/src/plugins/media/main/ts/core/Sanitize.ts:41]: We don't normally allow svg elements in the content and the tests all asserted that, however they also all had `on...` attributes.

Comment (lnewson) [reply modules/tinymce/src/plugins/media/main/ts/core/Sanitize.ts:41]: As discussed on slack, we'll stick with this and I'll add a test for this specifically.

File: modules/tinymce/changelog.txt [hunk 2]
```diff
@@ -3,6 +3,7 @@ Version 5.2.2 (2020-04-22)
     Fixed text decorations (underline, strikethrough) not consistently inheriting the text color #TINY-4757
     Fixed `format` menu alignment buttons inconsistently applying to images #TINY-4057
     Fixed the floating toolbar drawer height collapsing when the editor is rendered in modal dialogs #TINY-4837
+    Fixed the `media` plugin embed content not being processed safely in some cases #TINY-4857
```
Comment (tylerkelly13) [inline modules/tinymce/changelog.txt:6]: Fixed TinyMCE not processing embed content from the `media` plugin safely in some cases.

Maybe?
